Makefile

```
@@ -46,7 +46,8 @@ LDFLAGS := -X github.com//open-telemetry/opentelemetry-ebpf-profiler/vc.version=
 
 GO_FLAGS := -buildvcs=false -ldflags="$(LDFLAGS)" -tags osusergo,netgo
 
-all: generate ebpf binary
+all: generate ebpf
```

Can use `MAKEFLAGS` for this:

```suggestion
MAKEFLAGS += -j$(shell nproc)

all: generate ebpf
```

---

I thought about this. But this removes control from the user/developer who possibly only wants to do less parallel builds in case something else on the machine requires to get a certain amount of CPU. Maybe I am just biased, but in my past I *often* had to build applications on a running production machine (sounds awkward, but that's how reality plays sometimes).

---

How is it removing control when we're currently passing [-j$(shell nproc)](https://github.com/rockdaboot/opentelemetry-ebpf-profiler/blob/baf8db24eea59a77202443551f250cdfddcd3b3d/Makefile#L63) in the `ebpf` build (which is really where the parallel compilation via `Make` comes into play) ?

---

Just wanted to mention the Go argument and accept your suggestions.
I didn't know about `make -l`, sounds good to me :+1: